spec/javascripts/models/beta/story_spec.js

```diff
@@ -375,4 +375,123 @@ describe('Story model', function () {
       expect(changedStory.errors).toEqual([error]);
     });
   });
+
+  describe('isNew', () => {
+    it('returns true when story id is null', () => {
+      const story = { id: null }
+
+      expect(Story.isNew(story)).toBe(true);
+    });
+
+    it("returns false when story id isn't null", () => {
+      const story = { id: 42 }
+
+      expect(Story.isNew(story)).toBe(false);
+    });
+  });
+
+  describe('canDelete', () => {
+    it("returns true when story isn't accepted and isn't new", () => {
+      const story = { id: 42, state: 'started' };
+
+      expect(Story.canDelete(story)).toBe(true);
+    });
+
+    it('returns false when story is accepted', () => {
+      const story = { id: 42, state: 'accepted' };
+
+      expect(Story.canDelete(story)).toBe(false);
+    });
+
+    it('returns false when story is new', () => {
+      const story = { id: null, state: 'started' };
+
+      expect(Story.canDelete(story)).toBe(false);
+    });
+  });
+
+  describe('canSave', () => {
+    it("returns true when story isn't accepted and the text isn't empty", () => {
```

Comment:
Can we change from an `it` with a `'when'` in the description for a `describe` + an `it`? Like this:
```suggestion
    describe("when story isn't accepted and the text isn't empty", () => {
      it('returns true', () => {
```

Reply:
Sure :smile: